x-pack/test/api_integration/apis/cloud_security_posture/update_rules_config.ts

```diff
@@ -35,18 +35,22 @@ export default function ({ getService }: FtrProviderContext) {
       await esArchiver.unload('x-pack/test/functional/es_archives/fleet/empty_fleet_server');
     });
 
-    it(`Should return 500 when package policy id does not exist`, async () => {
+    it(`Should return 404 when package policy id does not exist`, async () => {
       const packagePolicyId = chance.guid();
 
       const { body: response } = await supertest
         .post(`/internal/cloud_security_posture/update_rules_config`)
         .set('kbn-xsrf', 'xxxx')
         .send({
           package_policy_id: packagePolicyId,
+          rules: [],
```

Comment:
we might want to extend the tests and pass some rules to change here and verify they changed in ES

Reply:
will do, but in a separate PR. 

i've written a [test](https://github.com/elastic/kibana/commit/243387167624a89f9ffd13e035c73222ac0764c9#diff-84ffd2c2036df35386f49f3f64617f93d38600c16eb3cbc13ad3d2d20e3c08b6R41-R62) to verify calling this API results with the correct changes in the package policy vars. 

it works, but.. 
1) the test is slow, about 18s, so i'm going to iterate some more on this. 
2) the entire test suite is currently flagged as flaky and after fixing it needs to be ran through the flaky test runner. i'm guessing it's because we used a hard-coded version number, which is fixed but also makes it slow 😄  

i'll open an issue to track this

Reply:
https://github.com/elastic/kibana/issues/140176